### ISSUE TITLE
Get the WCAG 2.2 testimonial video ready for publishing

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#4317]
## Brief
By the end of the cycle, the video should be ready for review by external parties.

We should have recorded the voiceover, overlayed this with the graphics and added an interview segment with a contributor.

## Epic lead
Kelly

## Driving role(s)
Charlotte

## Supporting roles
Mia, Trang

```[tasklist]
### Tasks
- [ ] https://github.com/alphagov/govuk-design-system/issues/4358
- [ ] https://github.com/alphagov/govuk-design-system/issues/4359
- [ ] https://github.com/alphagov/govuk-design-system/issues/4360
```
